extension/lib/LogHandler.jsm

```diff
@@ -45,22 +45,26 @@ this.LogHandler = {
     const timesinceLastUpload = Date.now() - lastUploadDate;
 
     if (timesinceLastUpload > Config.logSubmissionInterval) {
-      let entriesPingSize = Pioneer.utils.getEncryptedPingSize("online-news-log", 1, entries);
+      const entriesPingSize = Pioneer.utils.getEncryptedPingSize("online-news-log", 1, entries);
 
       if (entriesPingSize < UPLOAD_LIMIT) {
         // If the ping is small enough, just submit it directly
         await Pioneer.utils.submitEncryptedPing("online-news-log", 1, entries);
       } else {
         // Otherwise, break it into batches below the minimum size
-        let originalEntriesLength = entries.length;
+        const reduceRatio = UPLOAD_LIMIT / entriesPingSize;
+        const originalEntriesLength = entries.length;
+        let padding = INITIAL_PADDING;
```

Comment:
The reason `padding` was at the top scope was so that the log handler could learn what the right size was over time, instead of having to fail everytime if our padding guess is wrong.

Reply:
ah!! i'll change it back